htdocs/install/mysql/migration/13.0.0-14.0.0.sql

```diff
@@ -307,7 +307,8 @@ create table llx_product_perentity
     accountancy_code_sell_export  varchar(32),                        -- Selling accountancy code for vat export
     accountancy_code_buy          varchar(32),                        -- Buying accountancy code
     accountancy_code_buy_intra    varchar(32),                        -- Buying accountancy code for vat intracommunity
-    accountancy_code_buy_export   varchar(32)                  		-- Buying accountancy code for vat import
+    accountancy_code_buy_export   varchar(32),                     		-- Buying accountancy code for vat import
+    pmp double(24,8)
```

Comment:
The field must also be added into the llx_product_perentity.sql file so it will be availble for new installation too.